todo_app/templates/index.html

```diff
@@ -18,18 +19,19 @@ <h1 class="display-4">To-Do App</h1>
 	   </tr>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
-		  
+		
 		<table>
-		<h3><b>To do</h3></b>                                                                                                       
-		{%for x in my_objects: %}
+		<h3><b>To do</h3></b>    
```

Comment:
```suggestion
		<h3><b>To do</b></h3>
```
Tags need to be closed in the reverse order they are opened. `<h3>`, `<b>`, `<ul>`, `<table>` and `<tr>` all need matching closing tags.
This is confusing the browser which is then leading to issues in your e2e tests because the page isn't structured how you'd expect it.

A formatter is really helpful here, use the "Format Document" action (alt + shift + F on Windows) in VSCode.

Reply:
Updated,thank you